app/assets/javascripts/tylium/behaviors.js.coffee

```diff
@@ -193,3 +193,6 @@ document.addEventListener "turbolinks:load", ->
     history.pushState({}, '', this.href);
     $(this.hash)[0].scrollIntoView behavior: 'smooth'
     return
+
+  # Init Bootstrap tooltips
+  $('[data-toggle~=tooltip]').tooltip();
```

Comment:
We're already using tooltips why wasn't this needed before?

Reply:
It used to live in `app/assets/javascripts/tylium/modules/issues/evidence.js.coffee` and was moved here since it shouldn't be only for `issues#show`.